### PR TITLE
increase packaging taxonomization

### PR DESCRIPTION
**Pull request by teolemon**
### What
increase packaging taxonomization by adding XX: entries

### Part of
- #5520

### CODE REVIEW

File: taxonomies/packaging_materials.txt [hunk 1]
```diff
@@ -256,6 +257,7 @@ hu:Alkáli elem, 19 Alkáli, 19
 nl:Alkalinebatterij
 nl_be:Alkalinebatterij
 pt:Bateria alcalina, pilha alcalina, 19 Alkaline, 19
+xx:19 Alkaline, 19
```
Comment (stephanegigandet) [inline taxonomies/packaging_materials.txt:260]:
```suggestion
xx:19 Alkaline
```

plain numbers should not be added as synonym, that would trigger many false positives

File: taxonomies/packaging_materials.txt [hunk 2]
```diff
@@ -408,6 +419,7 @@ nl:Staal
 pt:Aço, 40 FE, RC 40, 40, RC Steel
 ru:Сталь
 ja:スチール
+xx:40 FE, Recycling Code 40, RC 40, 40, RC Steel
```
Comment (stephanegigandet) [inline taxonomies/packaging_materials.txt:422]:
```suggestion
xx:40 FE, Recycling Code 40, RC 40, RC Steel
```

File: taxonomies/packaging_materials.txt [hunk 3]
```diff
@@ -555,11 +581,13 @@ pt:Papel e plástico, Papel + plástico, 81 PapPet
 en:Biodegradable plastic, Card-stock Laminate, 87
 hu:Lebomló műanyag, Laminált kartonpapír, 87
 pt:Plástico biodegradável, 87
+xx:87
```
Comment (stephanegigandet) [inline taxonomies/packaging_materials.txt:584]: should be removed

Comment (teolemon) [inline taxonomies/packaging_materials.txt:584]:
```suggestion
```

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**